torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1818,74 +1818,95 @@ def sample_inputs_cat_concat(op_info, device, dtype, requires_grad, **kwargs):
     for input_shape1, input_shape2, kwargs in cases:
         yield SampleInput([make_arg(input_shape1), make_arg(input_shape2)], kwargs=kwargs)
 
-def error_inputs_cat(op_info, device, **kwargs):
-
+def error_inputs_cat(op_info, device, is_ref=False, **kwargs):
     make_arg = partial(make_tensor, device=device, dtype=torch.float32)
 
-    # error inputs for more than one element of the written-to tensor refer to a single memory location
-    yield ErrorInput(SampleInput([make_arg((S, S)), make_arg((S, S))],
-                                 kwargs={'out': make_arg((1, S)).expand((2 * S, S))}),
-                     error_regex='unsupported operation')
+    # TODO: primTorch has no checks for overlapping memory.
```

Comment:
What's our plan for this? Everything allocates/copies, but perhaps it might be important in some cases (when?).

Reply:
That's true, as we don't try to model the memory structure of tensors. This is a fine comment for now.